packages/teams-js/src/internal/internalAPIs.ts

```diff
@@ -8,12 +8,22 @@ import { compareSDKVersions } from './utils';
  * @internal
  * Limited to Microsoft-internal use
  */
-export function ensureInitialized(...expectedFrameContexts: string[]): void {
+export function ensureInitializeCalled(): void {
   if (!GlobalVars.initializeCalled) {
     throw new Error('The library has not yet been initialized');
   }
+}
+
+/**
+ * @internal
+ * Limited to Microsoft-internal use
+ */
+export function ensureInitialized(...expectedFrameContexts: string[]): void {
+  if (!GlobalVars.initializeCompleted || !GlobalVars.frameContext) {
+    throw new Error('The library has not yet been initialized');
```

Comment:
It might be nice to throw two different errors from this function. One for "Initialization hasn't begun yet" and another for "initialization has begun but hasn't finished yet"

Reply:
I tried that, but the amount of downstream issues that caused with UTs and E2E tests makes me think that's the kind of 'breaking change' we should avoid.